main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -38,32 +38,36 @@ public class FeedMetadata {
+  public Map<FeatureMetadata, Boolean> specFeatures = new LinkedHashMap<>();
 
   public ArrayList<AgencyMetadata> agencies = new ArrayList<>();
   private ImmutableSortedSet<String> filenames;
 
   public double validationTimeSeconds;
 
   // List of features that only require checking the presence of one record in the file.
-  private final List<Pair<String, String>> FILE_BASED_FEATURES =
+  private final List<Pair<FeatureMetadata, String>> FILE_BASED_FEATURES =
       List.of(
-          new Pair<>("Pathways (basic)", GtfsPathway.FILENAME),
-          new Pair<>("Transfers", GtfsTransfer.FILENAME),
-          new Pair<>("Fares V1", GtfsFareAttribute.FILENAME),
-          new Pair<>("Fare Products", GtfsFareProduct.FILENAME),
-          new Pair<>("Shapes", GtfsShape.FILENAME),
-          new Pair<>("Frequencies", GtfsFrequency.FILENAME),
-          new Pair<>("Feed Information", GtfsFeedInfo.FILENAME),
-          new Pair<>("Attributions", GtfsAttribution.FILENAME),
-          new Pair<>("Translations", GtfsTranslation.FILENAME),
-          new Pair<>("Fare Media", GtfsFareMedia.FILENAME),
-          new Pair<>("Zone-Based Fares", GtfsArea.FILENAME),
-          new Pair<>("Transfer Fares", GtfsFareTransferRule.FILENAME),
-          new Pair<>("Time-Based Fares", GtfsTimeframe.FILENAME),
-          new Pair<>("Levels", GtfsLevel.FILENAME),
-          new Pair<>("Booking Rules", GtfsBookingRules.FILENAME),
-          new Pair<>("Fixed-Stops Demand Responsive Transit", GtfsLocationGroups.FILENAME));
+          new Pair<>(new FeatureMetadata("Pathway Connections", "Pathways"), GtfsPathway.FILENAME),
+          new Pair<>(new FeatureMetadata("Levels", "Pathways"), GtfsLevel.FILENAME),
+          new Pair<>(new FeatureMetadata("Transfers", null), GtfsTransfer.FILENAME),
+          new Pair<>(new FeatureMetadata("Shapes", null), GtfsShape.FILENAME),
+          new Pair<>(new FeatureMetadata("Frequencies", null), GtfsFrequency.FILENAME),
+          new Pair<>(new FeatureMetadata("Feed Information", null), GtfsFeedInfo.FILENAME),
+          new Pair<>(new FeatureMetadata("Attributions", null), GtfsAttribution.FILENAME),
+          new Pair<>(new FeatureMetadata("Translations", null), GtfsTranslation.FILENAME),
+          new Pair<>(new FeatureMetadata("Fares V1", "Fares"), GtfsFareAttribute.FILENAME),
+          new Pair<>(new FeatureMetadata("Fare Products", "Fares"), GtfsFareProduct.FILENAME),
+          new Pair<>(new FeatureMetadata("Fare Media", "Fares"), GtfsFareMedia.FILENAME),
+          new Pair<>(new FeatureMetadata("Zone-Based Fares", "Fares"), GtfsArea.FILENAME),
+          new Pair<>(
+              new FeatureMetadata("Fares Transfers", "Fares"), GtfsFareTransferRule.FILENAME),
+          new Pair<>(new FeatureMetadata("Time-Based Fares", "Fares"), GtfsTimeframe.FILENAME),
+          new Pair<>(
+              new FeatureMetadata("Booking Rules", "Flexible Services"), GtfsTimeframe.FILENAME),
```

Comment:
Shouldn't the parameter be GtfsBookingRules.FILENAME
and GtfsLocationGroups.FILENAME for the next line?